dockers/docker-orchagent/orchagent.sh

```diff
@@ -61,7 +61,7 @@ elif [ "$platform" == "barefoot" ]; then
 elif [ "$platform" == "vs" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
 elif [ "$platform" == "mellanox" ]; then
-    ORCHAGENT_ARGS+=""
+    ORCHAGENT_ARGS+="-k 10000"
```

Comment:
I would like to understand what are the potential impact following this change beside the performance improvement with routes config

Reply:
Bulk size determines the objects count we send to SAI when calling to SAI API.
Bulk size is used in SONiC for sai_route_api and for sai_next_hop_group_api.
In SAI implementation, the handler is using the object count as the for-loop size when calling to the relevant SDK API.
It basically means how many times we will call SDK API for a single SAI API call.

Reply:
@liorghub What is the motivation for this change?

Reply:
Using the default bulk size (1K) gave poor results, the improvement in routes config time was very low. 
In a debug session held with SDK, Jinu instructed to increase the bulk size and it worked.
This is the explanation I got from him.
By it's nature the async works well when it is like a pipeline. i.e., by having a new request to process when the last one is already done. This keeps the threads always working and gives the best performance. 
When you break the bulk into smaller chunks it is possible that a sync API like an internal event for example gets pushed into the queue which breaks the flow.
Essentially the async router and the other APIs work on different threads.
When you have other APIs, either user provided or internal come in, the Async pipeline will essentially stall until the sync API is complete. Having a continuous stream of async APIs keeps the threads active and allows for the best parallel processing.

Reply:
@stepanblyschak as discussed, I removed the bulk size change.